processor/routingprocessor/README.md

```diff
@@ -0,0 +1,37 @@
+# Routing processor
+
+Routes traces to specific exporters.
+
+This processor will read a header from the incoming HTTP request (gRPC or plain HTTP) and direct the trace information to specific exporters based on the attribute's value.
+
+This processor *does not* let traces to continue through the pipeline and will emit a warning in case other processor(s) are defined after this one. Similarly, exporters defined as part of the pipeline are not authoritative: if you add an exporter to the pipeline, make sure you add it to this processor *as well*, otherwise it won't be used at all. All exporters defined as part of this processor *must also* be defined as part of the pipeline's exporters.
+
```

Comment:
We also need to tell that this processor cannot be used with `batch` and others which do not keep the incoming context, right?

Reply:
Done!